### DISCUSSION TITLE
Quarkus 3.3.1 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#35705] [Quarkus Blog/Website]
# Quarkus 3.3.1 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-3-1-released/

**Comment by Naresh1143** [comment]
Hi
Thanks for sharing this information, this blog was gives great information and it's is really the best medical device and it will very useful for the Therapeutic patients thanks for sharing this this information.
Here is sharing some Oracle Fusion Cloud Technical information may be its helpful to you.
<a href="https://www.gologica.com/course/oracle-fusion-cloud-technical-training/"rel="nofollow">Oracle Fusion Training Cloud Technical</a>
